sqlmesh/integrations/github/cicd/controller.py

```diff
@@ -468,7 +469,8 @@ def get_plan_summary(self, plan: Plan) -> str:
                 return "No changes to apply."
             return f"{difference_summary}\n{missing_dates}"
         except PlanError as e:
-            return f"Plan failed to generate. Check for pending or unresolved changes. Error: {e}"
+            err_str = f" Error: {e}" if not isinstance(e, PlanApplyError) else ""
```

Comment:
The `Error:` prefix doesn't really look useful. The first sentence already indicates that an error occurred